### PR TITLE
(plugin) database::mysql - add nlabel to mysql modes

### PR DESCRIPTION
**Pull request by lchrdn**
Adding nlabel to mysql modes

### CODE REVIEW

File: database/mysql/mode/innodbbufferpoolhitrate.pm [hunk 1]
```diff
@@ -103,12 +103,16 @@ sub run {
         $self->{output}->output_add(severity => $exit_code,
                                     short_msg => sprintf("innodb buffer pool hitrate at %.2f%%", $prcts{'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now')})
                                     );
-        $self->{output}->perfdata_add(label => 'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), unit => '%',
+        $self->{output}->perfdata_add(label => 'bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), 
+                                      nlabel => 'database.bufferpool_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now') . '.percentage',
```
Comment (garnier-quentin) [inline database/mysql/mode/innodbbufferpoolhitrate.pm:107]: Need to talk it the naming for that one (kind of particular with the now).

File: database/mysql/mode/passwordexpiration.pm [hunk 2]
```diff
@@ -45,13 +45,13 @@ sub set_counters {
...
     $self->{maps_counters_type} = [
-        { name => 'users', type => 2, format_output => '%s user(s) detected', display_counter_problem => { label => 'users', min => 0 },
+        { name => 'users', type => 2, format_output => '%s user(s) detected', display_counter_problem => { label => 'users', nlabel => 'database.users.count', min => 0 },
           group => [ { name => 'user', skipped_code => { -11 => 1 } } ] 
         }
     ];
 
     $self->{maps_counters}->{user} = [
-        { label => 'status', type => 2, critical_default => '%{expire} ne "never" and %{expire_time} == 0', set => {
+        { label => 'status', nlabel => 'database.users.password.status', type => 2, critical_default => '%{expire} ne "never" and %{expire_time} == 0', set => {
```
Comment (garnier-quentin) [inline database/mysql/mode/passwordexpiration.pm:54]: there is no nlabel for type 2 (status).

File: database/mysql/mode/slowqueries.pm [hunk 3]
```diff
@@ -98,6 +98,7 @@ sub run {
                                         ($result - $old_val), ($new_datas->{last_timestamp} - $old_timestamp), $value)
                                     );
         $self->{output}->perfdata_add(label => 'slow_queries_rate',
+                                      nlabel => 'database.slowqueries.persecond',
```
Comment (garnier-quentin) [inline database/mysql/mode/slowqueries.pm:101]: maybe: database.queries.slow.persecond

File: database/mysql/mode/innodbbufferpoolhitrate.pm (hunk 1 above)
Comment (Sims24) [reply database/mysql/mode/innodbbufferpoolhitrate.pm:107]: As discussed, delta seems legit

File: database/mysql/mode/passwordexpiration.pm (hunk 2 above)
Comment (Sims24) [reply database/mysql/mode/passwordexpiration.pm:54]: +1

File: database/mysql/mode/qcachehitrate.pm [hunk 4]
```diff
@@ -120,12 +120,16 @@ sub run {
         $self->{output}->output_add(severity => $exit_code,
                                     short_msg => sprintf("query cache hitrate at %.2f%%", $prcts{'qcache_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now')})
                                     );
-        $self->{output}->perfdata_add(label => 'qcache_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), unit => '%',
+        $self->{output}->perfdata_add(label => 'qcache_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now'), 
+                                      nlabel => 'database.qcache_hitrate' . ((defined($self->{option_results}->{lookback})) ? '' : '_now') . '.percentage',
```
Comment (garnier-quentin) [inline database/mysql/mode/qcachehitrate.pm:124]: maybe: database.qcache.hitrate

**Review by garnier-quentin — APPROVED** [review]

**Review by garnier-quentin — APPROVED** [review]

**Pull request merged**